### PR TITLE
Add Get-BitlockerInformation

### PR DESCRIPTION
**Pull request by andrewbaker-uk**
Uploading code, readme and scan profile

### CODE REVIEW

File: PowerShell Scanners/Get Bitlocker information/Bitlocker Information.ps1 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+if (-Not(Get-Command Get-BitlockerVolume -ErrorAction SilentlyContinue)) {
+    throw "Unable to find Get-BitlockerVolume. Available on Windows 10 or Server 2016 with the Bitlocker feature installed"
+}
```
Comment (PDQDakota) [inline PowerShell Scanners/Get Bitlocker information/Bitlocker Information.ps1]: Could you change the if block over to? `#Requires -Modules BitLocker`

**Review by PDQDakota — CHANGES_REQUESTED** [review]
Looks good! Just that one change and I'll bring it right on in.

File: PowerShell Scanners/Get Bitlocker information/Bitlocker Information.ps1 (hunk 1 above)
Comment (andrewbaker-uk) [reply PowerShell Scanners/Get Bitlocker information/Bitlocker Information.ps1]: Sure

**Review by PDQDakota — APPROVED** [review]
Thank you so much!

**Review by Colby-PDQ — APPROVED** [review]

**Pull request merged**